### PR TITLE
QUEST_LV_0100 : (Various fixes, storyline: Gesti till a portion of Tiltas Valley)

### PR DESCRIPTION
**Pull request by Sunaries**
Minor updates for the Gesti storyline.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -6812,7 +6812,7 @@ QUEST_LV_0100_20150428_006811	Meet Blacksmith of Klaipeda
 QUEST_LV_0100_20150428_006812	Go meet the Blacksmith of Klaipeda as Pipoti has told you.
 QUEST_LV_0100_20150428_006813	Before you release the important seal at Ramstis Valley, you should first talk to the disciple of Gustas.
 QUEST_LV_0100_20150428_006814	As the Goddess told you in the dream of the bishop, you should first go to Crystal Mine, but you can't just see and ignore the villagers who are evacuating. Let's help the angry hunter.
-QUEST_LV_0100_20150428_006815	Talk to Bidotas at the 1st floor of Crystal Mine
+QUEST_LV_0100_20150428_006815	Talk to Vaidotas on the 1st floor of Crystal Mine
```
Comment (ttgmichael) [inline QUEST_LV_0100.tsv]: Talk to Vaidotas in Crystal Mine 1F? :P

**Comment by Sunaries** [comment]
Good to merge.

**Comment by terrapin47** [comment]
I've created a pull request to your branch, which will resolve the merge conflicts.

https://github.com/Sunaries/EnglishTranslation/pull/2

**Comment by imcgames** [comment]
I'll wait for the conflicts to be resolved before merging this PR :wink: 
Thanks @zhouyu47

**Comment by Sunaries** [comment]
@imcgames You should be able to merge this pull request successfully now. :)

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**